backend/src/apiserver/config/sample_config.json

```diff
@@ -1,32 +1,32 @@
 [
   {
-    "name":"[Sample] ML - XGBoost - Training with Confusion Matrix",
-    "description":"A trainer that does end-to-end distributed training for XGBoost models. For source code, refer to https://github.com/kubeflow/pipelines/tree/master/samples/core/xgboost_training_cm",
-    "file":"/samples/core/xgboost_training_cm/xgboost_training_cm.py.tar.gz"
+    "name": "[Sample] ML - XGBoost - Training with Confusion Matrix",
+    "description": "A trainer that does end-to-end distributed training for XGBoost models. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/core/xgboost_training_cm)",
+    "file": "/samples/core/xgboost_training_cm/xgboost_training_cm.py.tar.gz"
   },
   {
-    "name":"[Sample] Basic - Sequential execution",
-    "description":"A pipeline with two sequential steps. For source code, refer to https://github.com/kubeflow/pipelines/blob/master/samples/core/sequential/sequential.py",
-    "file":"/samples/core/sequential/sequential.py.tar.gz"
+    "name": "[Sample] Basic - Sequential execution",
+    "description": "A pipeline with two sequential steps. [source code](https://github.com/kubeflow/pipelines/blob/master/samples/core/sequential/sequential.py)",
+    "file": "/samples/core/sequential/sequential.py.tar.gz"
   },
   {
-    "name":"[Sample] Basic - Parallel execution",
-    "description":"A pipeline that downloads two messages in parallel and prints the concatenated result. For source code, refer to https://github.com/kubeflow/pipelines/blob/master/samples/core/parallel_join/parallel_join.py",
-    "file":"/samples/core/parallel_join/parallel_join.py.tar.gz"
+    "name": "[Sample] Basic - Parallel execution",
+    "description": "A pipeline that downloads two messages in parallel and prints the concatenated result. [source code](https://github.com/kubeflow/pipelines/blob/master/samples/core/parallel_join/parallel_join.py)",
```

Comment:
I believe XGBoost sample also needs Permission Requirement link. @numerology who owns this sample

Reply:
Thanks for bringing this up. Yes it requires dataproc permission.

Reply:
XGboost sample needs storage.admin permission as well

Reply:
Is this described in XGBoost README file? @gaoning777 has a PR for it, I remember.
Yuan, you can just point to README file.

Reply:
Yes @gaoning777 made #2359 for this.

Reply:
Updated link